python/sglang/bench_offline_throughput.py

```diff
@@ -0,0 +1,160 @@
+"""
+Benchmark the throughput of using the offline LLM engine.
+This script does not launch a server.
+It accepts the same arguments as bench_latency.py
+
+# Usage
+python -m sglang.bench_offline_throughput --model-path meta-llama/Meta-Llama-3-8B-Instruct --batch 1 12 14 --input-len 256 512 --output-len 32 256 --result-filename out.jsonl
+
+"""
+
+import argparse
+import dataclasses
+import itertools
+import logging
+import time
+from typing import Dict, List, Tuple
+
+import jsonlines
+
+from sglang.api import Engine as getEngine
+from sglang.srt.server import Engine
+from sglang.srt.server_args import ServerArgs
+
+
+@dataclasses.dataclass
+class BenchArgs:
+    run_name: str = "before"
+    batch_size: Tuple[int] = (1,)
+    input_len: Tuple[int] = (1024,)
+    output_len: Tuple[int] = (16,)
+    result_filename: str = ""
+    # Plotting args
+    graph_sql: str = (
+        "select run_name, batch_size, prefill_throughput from results where run_name='before'"
+    )
+    graph_filename: str = "out.png"
```

Comment:
With my comment above, we can probably remove all of these arguments. 
Remove `graph_sql` because it is not used.